internal/http/router.go

```diff
@@ -120,6 +123,27 @@ func getSrv(r *router.Router) fasthttp.Server {
 // Serve starts our HTTP server and request router
 func Serve() error {
 	log = config.GetLogger()
+
+	if config.UseCustomHeffalump {
+		content, err := os.ReadFile(config.BookFilename)
```

Comment:
oooooooh no, we definitely want to load this up ahead of time and store it in memory instead of doing disk I/O every time we serve a request 😬